patterns/byod.md

```diff
@@ -0,0 +1,579 @@
+
+#### Azure AD Identity Protection
+
+Azure AD Identity Protection is an Azure AD feature that protects the identity through:
+
+* automation of detection and remediation of identity-based risks,
+* includes additional challenge to user based on risk detection in real time, and
+* Provides a portal for identify based investigation.
+
+#### Defender for Cloud Apps
+
+Defender for Cloud Apps (formerly Microsoft Cloud App Security (MCAS)) provides capabilities that can restrict access to only authorised platforms and device types and can minimise the risk of data loss and spills within a supported app or browser. Defender for Cloud App Session Control is enabled through Conditional Access policies (App Control), which enables Defender for Cloud App to act as a Cloud Access Security Broker (CASB)) for the connection from the browser to the data source. Further control through the CASB can:
+
+* Block data upload/download
+* Check device characteristics
+* Block printing
+* Block copy/paste actions
+
+#### Implementation Decisions
+
+The following tables describe the high level implementation decisions
+
+Decision Point | Design Decision | Justification
+--- | --- | ---
+Mobile device use cases | Authorised applications on iOS and Android devices without MDM enrollment (MAM) | Allowing MAM on these device platforms offers great flexibility but also greater risk than full application access on managed Windows devices. Less risk can be provided by the virtual desktop BYOD option. 
+Conditional Access | Conditional Access will be configured with exception group for those trusted for MAM | Provides a Zero Trust method to restrict access to this use case only and allows Protected Utility Blueprint baseline configuration for remainder of Agency that do not require BYOD for their persona. 
+Defender for Cloud Apps | Cloud App security Session policies will restrict access to data within browser sessions | Provides a Zero Trust method to restrict access to leaking of data. 
+Intune app protection policies | App protection polices will be deployed to the MAM identity groups without enrollment required (MAM-WE) | Provides some protection against Agency data within a managed app. Protection will require a passcode within the app, control sharing of data between apps and prevent copy of data to personal locations. This allows separation between corporate and personal data to address the intent of ISM security control 1400. 
+Azure AD Identity Protection | Blueprint configuration will remain in place enabling Enable the sign-in risk policy and user risk policy within the Azure AD tenant | Provide reporting of detected suspicious sign-in activity based on defined MFA, sign-in risk and user risk policies for increased security and automated remediation steps such as requiring a password change or MFA challenge when authentication is suspicious. 
```

Comment:
`Blueprint configuration will remain in place enabling Enable the sign-in risk policy and user risk policy within the Azure AD tenant`

Maybe some missing code blocks or is it bad sentence joining?